lib/rules/sort-imports.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * @fileoverview Rule to require sorting of import declarations
+ * @author Christian Schuller
```

Comment:
Please add copyright and license. See http://eslint.org/docs/developer-guide/code-conventions.html#file-format